src/components/KCollapse/KCollapse.vue

```diff
@@ -1,30 +1,28 @@
 <template>
-  <div class="k-collapse w-100">
+  <div class="k-collapse">
     <div
-      class="k-collapse-heading mb-3"
-      :class="trailingTrigger ? 'd-flex' : 'd-block'"
+      class="k-collapse-heading"
+      :class="trailingTrigger ? 'k-collapse-heading-flex' : 'k-collapse-heading-block'"
```

Comment:
Instead of creating sub-classes for different variants, why don't you create a class `.has-trailing-trigger` (or something along those lines) and use combination of default styles (that would apply to `.k-collapse-heading`) and extend it when needed (through `.k-collapse-heading.has-trailing-trigger`)?

It would look something like:
```
:class="{ 'has-trailing-trigger': trailingTrigger }"
```

Then in your css:
```
.k-collapse-heading {
    display: flex !important;
    
    .k-collapse-title {
        margin-bottom: ... // here you can also nest default styles that apply to `.k-collapse-title` instead of creating `k-collapse-title-margin-right` since the condition is the same (`!trailingTrigger`)
    }
    ...
    &.has-trailing-trigger {
        display: block !important;
        
        .k-collapse-title {
            margin-right: ... // here you can also nest styles that apply to `.k-collapse-title` instead of creating `k-collapse-title-margin-right` since the condition is the same (`trailingTrigger === true`)
        }
    }
}
```